extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -225,9 +225,15 @@ export class AmpAnalytics extends AMP.BaseElement {
         this.instrumentation_.createAnalyticsGroup(this.element);
 
     if (this.config_['transport'] && this.config_['transport']['iframe']) {
+      const ampAdResourceId = getAmpAdResourceId(this.element, this.win.top);
```

Comment:
Consider using user().assert here as well

Reply:
Done.
Also corrected wording of error message.